contrib/platform/test/com/sun/jna/platform/win32/Kernel32Test.java

```diff
@@ -539,21 +539,6 @@ public void testCreateProcess() {
         assertTrue(processInformation.dwProcessId.longValue() > 0);
     }
 
-    public void testGetEnvironmentVariable() {
```

Comment:
This I can live with - may I suggest then the following: accept the suggested pull request and merge it into the _master_, then simply go to the (3) classes _NamedPipeTest_, _EnvironmentVarsTest_ and _WinconTest_ that I submitted so far and renamed them (5 seconds effort). This will serve both as an example and a reminder for future developers (including myself)